### DISCUSSION TITLE
Error when following guide on installing 0.111.0

### DISCUSSION THREAD

**Discussion by jullle3** [fastapi/fastapi#11595] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
N/A
```

### Description

When in follow the install guide https://fastapi.tiangolo.com/#installation it shows as if i can just write "pip install fastapi" and it will install everything by itself, however when running `pip3 install fastapi~=0.111.0` i get an error indicating problems with httptools. Theres no mention in the docs for anything regarding this. I found fixes on google showing i should install python3-dev tools etc etc, and if that is in fact true, the docs should include it. My solution was to install fastapi 0.110.0 instead, which worked flawlessly. 

```
      httptools/parser/parser.c:20:10: fatal error: Python.h: No such file or directory
         20 | #include "Python.h"
            |          ^~~~~~~~~~
      compilation terminated.
      error: command '/usr/bin/x86_64-linux-gnu-gcc' failed with exit code 1
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for httptools
  Running setup.py clean for httptools
Failed to build httptools
ERROR: Could not build wheels for httptools, which is required to install pyproject.toml-based projects

```

### Operating System

Linux

### Operating System Details

Ubuntu 20.04

### FastAPI Version

0.111.0

### Pydantic Version

the default

### Python Version

3.11.9

**Comment by mattmess1221** [comment]
Do you not have wheel installed? You should probably be using a virtualenv since it creates a linked python install independent from the system. Make sure you install `python3-venv`.

```
python3 -m venv .venv
source .venv/bin/activate
pip install fastapi
```

**Reply by jullle3** [reply]
I have wheel 0.41.2 installed and am running inside a venv (automatically setup by intelliJ) 

Still errors out. Does yours work? 

@killjoy1221

**Comment by YuriiMotov** [comment]
This has been updated, in current version of docs the command to install is  `pip install "fastapi[standard]"`.
